apps/reactlit-examples/src/pages/hello-world-vanilla/index.tsx

```diff
@@ -1,32 +1,66 @@
 import { Reactlit, useReactlitState } from '@reactlit/core';
 import { configureInputs } from '@reactlit/vanilla';
 
-const Inputs = configureInputs({
-  wrapper: (children) => (
-    <div className="flex gap-2 items-center">
-      <label htmlFor="name-input">Name</label>
-      {children}
-    </div>
-  ),
-});
+const Inputs = (label: string) =>
+  configureInputs({
+    wrapper: (children) => (
+      <div className="flex gap-2 items-center">
```

Comment:
We have the labels set up within the individual components, so I think it would be better to leave them there and specify via the `label` prop instead of using the wrapper. I think in general it will be cleaner to use inputs in a similar recommended way like observable does where it's always a constant you can reference with `Inputs.Text` or `Inputs.Check` instead of having the `Inputs()` then the config routine would get called fresh on every render.